extensions/sonarr/README.md

```diff
@@ -0,0 +1,3 @@
+# Sonarr
+
+View your Sonarr instance, upcoming shows and much more
```

Comment:
Since the user needs to setup some preferences before using the extension it would be good to add some instructions on how to do so. Like get the API Key, which host and port to use, etc.

Reply:
Yes I know, that this will happen for most users. But it is always nice at least to tell them where they can find the API key within Sonarr and also something we do with every other extension that requires some configuration

Reply:
Understandable!! I've added instructions on how to retrieve API key from sonarr 🎉